### ISSUE TITLE
Images as part of push message

### ISSUE THREAD

**Issue by saifkamaal** [zo0r/react-native-push-notification#742]
Hi,

Using this API can we show images(big images/small images) received as part of push messages?

Something like this ?
![screenshot_2018-05-17-12-20-14-939_com miui home](https://user-images.githubusercontent.com/29369768/40169148-6153e6b8-59d6-11e8-934b-c81b9af225e7.png)

**Comment by leventyildirim61** [comment]
Hello.

it is solved #668. Please check this. I think that it's useful for you. @saifkamaal

**Comment by theyogendraDanwar** [comment]
@leventyildirim61 I don't think the repo is updated with the recent code imageUrl option is not coming for the localNotification method

**Comment by theyogendraDanwar** [comment]
Can somebody please guide me how do I achieve an image with the notification. I am getting the image when the app is in the background but I want to use localNotification to push image with text to the notification system.

**Comment by Dallas62** [comment]
@theyogendraDanwar This behaviour is not implemented yet, there is #668 that implement a solution, but I didn't see an iOS implementation of this on [react-native-community/push-notification-ios](https://github.com/react-native-community/push-notification-ios).

**Comment by DanGDroid** [comment]
@theyogendraDanwar 
on Android i successfully implemented support for image
but for ios i cannot show a local notification at all
note: i am using react-native-firebase/messaging `onMessage()` to listen for remote notifications while in Foreground

```
function createLocalNotification(message) {
  console.log(
    'PushNotificationSagas.js Notifications createLocalNotification message ',
    message,
  );
  const { notification, data, messageId } = message || {};
  if (_.isEmpty(notification)) {
    console.log(
      'PushNotificationSagas.js Notifications createLocalNotification message is empty -- continue',
    );
    return;
  }
  PushNotification.localNotification({
    title: notification.title,
    message: notification.body,
    messageId: messageId,
    data: data,
    color: '#e9632e',
    channelId: ANDROID_PUSH_CHANNEL_ID,
    smallIcon: '@mipmap/ic_notification',
    bigPictureUrl: notification?.android?.imageUrl,
    largeIconUrl: notification?.android?.imageUrl,
  });
}
